lib/rpmrc.c

```diff
@@ -1103,6 +1111,12 @@ static void defaultMachine(rpmrcCtx ctx, const char ** arch, const char ** os)
 #		endif
 #	endif
 
+#if defined(__linux__)
+	/* in linux, lets rename parisc to hppa */
+	if (rstreq(un.machine, "parisc"))
+	    strcpy(un.machine, "hppa");
```

Comment:
This feels like an unrelated change. As far as I know the Crusoe has nothing to do with `hppa`...